pyquil/api/_compiler.py

```diff
@@ -135,6 +135,7 @@ def native_quil_to_executable(
     def _fetch_calibration_program(self) -> Program:
         response = get_quilt_calibrations(
             quantum_processor_id=self.quantum_processor_id,
+            client=self.client_configuration,
```

Comment:
The attribute is stored on the class with an underscore: 
```suggestion
            client=self._client_configuration,
```